config/laravellocalization.php

```diff
@@ -32,20 +42,60 @@
             'regional'     => 'es_ES',
             'locale_win'   => 'Spanish_Spain.1252',
             'date_formats' => [
-                'default' => 'd/m/Y h:i:s',
+                'default' => 'j/m/Y H:i:s',
+            ],
+        ],
+        'de' => [
```

Comment:
It seems to be too early to add translation to configs and seeders, without having translation files

Reply:
> It seems to be too early to add translation to configs and seeders, without having translation files

If it's so important, there are my version of translation frontend lines for that languages: https://github.com/Limych/laravel-boilerplate/commit/cd2c573dfffe7bd8196303f44d931e8550203579